react/scss/styles.scss

```diff
@@ -0,0 +1,114 @@
+.bookSearch {
+  max-width: 792px;
+  margin: 0 auto;
+}
+
+.bookItem {
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 120px;
+  height: 232px;
+  margin: 35px;
+  padding: 3px;
+  cursor: pointer;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  position: relative;
+
+  &:hover:before{
+    border: 3px solid $green;
+    border-radius: 4px;
+    display: inline-block;
+    content: '';
+    position: absolute;
+    top: 0; right: 0; bottom: 0; left: 0;
+    z-index: -1;
+  }
+}
+
+.bookList {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.bookTitle {
+  width: 100%;
+  font-family: $font;
+  font-size: $font-medium;
+  letter-spacing: 0px;
+  line-height: 19px;
```

Comment:
I think this `line-height` should belong to a variable - Although I think this is not needed 🤔